### PR TITLE
cmake: added simple test

### PR DESCRIPTION
**Pull request by junghans**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -14,16 +14,20 @@ env:
     - TESTING=ON TESTOPTS="-E \(imc\|spce_cma_simple\)" CMAKE_BUILD_TYPE=Debug
     - TESTING=OFF CMAKE_BUILD_TYPE=Debug   WERROR=yes
     - TESTING=ON TESTOPTS="-E \(imc\|spce_cma_simple\)" CMAKE_BUILD_TYPE=Release DOCKERHUB=yes
+    - DISTRO=fedora_gmx5.1 TESTING=ON TESTOPTS="-E \(imc\|spce_cma_simple\)" CMAKE_BUILD_TYPE=Release
```
Comment (JensWehner) [inline .travis.yml]: is spce part of csg?

Comment (junghans) [inline .travis.yml]: Yes, but all tests get executed all the time! (I am sick of maintaining 5 versions of `.travis.yml`!)

Comment (JensWehner) [inline .travis.yml]: fair point.

Comment (junghans) [inline .travis.yml]: We could label the test here a xtp-test:
```cmake
SET_TESTS_PROPERTIES( ${PROG}Help PROPERTIES LABEL "xtp" )
```
and well as `votca` and then use `TESTOPTS="-L xtp"` to only run those...

**Comment by junghans** [comment]
@JensWehner : ready to merge.

File: src/tools/CMakeLists.txt [hunk 2]
```diff
@@ -21,5 +21,11 @@ foreach(PROG xtp_map xtp_run xtp_tools xtp_parallel xtp_dump)
      install(FILES ${CMAKE_CURRENT_BINARY_DIR}/${PROG}.man DESTINATION ${MAN}/man1 RENAME ${PROG}.1)
   endif (BUILD_MANPAGES)
   list(APPEND XTP_PROGS "${PROG}")
+
+  if(ENABLE_TESTING)
+    add_test(${PROG}Help ${PROG} --help)
+    # run tests for tools, csg, moo, ctp as well for coverage
+    set_tests_properties(${PROG}Help PROPERTIES LABELS "xtp;tools;csg;ctp;moo;votca")
```
Comment (junghans) [inline src/tools/CMakeLists.txt]: Logic here is a bit backwards. One basically has to list all package, which should run the local tests.
E.g. a commit in `tools` should break `xtp`.

Comment (JensWehner) [inline src/tools/CMakeLists.txt]: ahh okay I see.

File: src/tools/CMakeLists.txt [hunk 3]
```diff
@@ -21,5 +21,11 @@ foreach(PROG xtp_map xtp_run xtp_tools xtp_parallel xtp_dump)
      install(FILES ${CMAKE_CURRENT_BINARY_DIR}/${PROG}.man DESTINATION ${MAN}/man1 RENAME ${PROG}.1)
   endif (BUILD_MANPAGES)
   list(APPEND XTP_PROGS "${PROG}")
+
+  if(ENABLE_TESTING)
+    add_test(${PROG}Help ${PROG} --help)
```
Comment (junghans) [inline src/tools/CMakeLists.txt]: @baumeier: This is the cmake way to run test with `make test` (when build with `-DENABLE_TESTING=ON`).

Currently the test are just `PROGRAM --help`, but this is meant as a starting point.

**Pull request merged**